src/index.js

```diff
@@ -23,15 +23,21 @@ var args = minimist(process.argv.slice(2), {
     maxjobs: '1000',            // max number of jobs
     geometry: false,            // retrieve geometry (-g)
     router: "osrm",             // routing backend
-    router_address: "0.0.0.0",
-    router_port: 5000,
     override: true,             // allow cl option override (-g only so far)
     logdir: __dirname + '/..',  // put logs in there
     limit: '1mb',               // max request size
     timeout: 5 * 60 * 1000      // milli-seconds.
   }
 });
 
+// For each routing profile (e.g., car) add a host and a port.
+var routingServers = {
+  'car': {
+    'host': '0.0.0.0',
+    'port': '8082'
```

Comment:
Thanks for the update! I'd rather stick with the previous port 5000 here so that default behaviour is unchanged and works out-of-the-box with OSRM. Otherwise reading to merge. :+1:

Reply:
Oh sorry, I forgot to revert my local change.